index.html

```diff
@@ -1,158 +1,132 @@
-            <li>Погружение в HTML-разметку
-                <ul>    
-                    <li>Мета-теги.</li>
-                    <li>Мнемоники.</li>
-                    <li>Тег для работы с изображениями.</li>
-                    <li>Теги для работы со списками.</li>
-                    <li>Теги для работы с таблицами.</li>
-                    <li>Теги для работы с формами.</li>
-                    <li>Div-блоки.</li>
-                </ul>    
-             </li>
-          
-    </ol>    
+    <header>
+
+        <div class="container">
+            <div class="header__top">    
+                <div class="header__top-inner">
+                    <div class="logo"></div>
+                    <nav>
+                        <ul class="nav__list">                   
+                            <li class="nav__list-item">   
+                                <a class="active__link" href="#">Stays</a>
+                            </li>
+                            <li class="nav__list-item">
+                                <a class="menu__link" href="#">Attractions</a>
+                            </li>   
+                            <li class="nav__list-item">
+                                <a class="menu__link" href="#"></a>
+                            </li>
+                            <li class="nav__list-item">
+                                <a class="menu__link" href="#"></a>
+                            </li>
+                        </ul>
+                    </nav>
+                </div>
+            </div> 
+            
+            <div class="header__content">
+                <h1>Discover stays <span> to live,work or just relax</span></h1>
+                <form action="/">
+                    <div class="header__input--search">
+                        <p>
+                            <label for="search">Your destination or hotel name</label>
+                        </p>
+                        <input type="text" name="destination" id="destination" placeholder="New York">
+                    </div>
+                    <div class="header__input--check">
+                        <p>
+                            <label for="in">Check-in - Check-out</label>
+                        </p>
+                        <input type="text" name="date" id="date" placeholder="Tue 15 Sept &ndash; Sat 19 Sept" onfocus="(this.type='date')" onblur="(this.type='text')">
+                    </div>
+                    <div class="header__input--persons">
+                        <input class="persons" placeholder="2 Adults &ndash; 0 Children &ndash; 1 Room"/>   
+                    </div>
+                    <button class="header__btn">Search</button>
+                </form>
+                
+                <div class="markets">
+                    <div class="android">
+                        <img src="/img/google-play.svg" alt="play market">
+                    </div>
+                    <div class="ios">
```

Comment:
what the ios ? let's use more better naming here. 
BTW: Agree with @Dronom regarding unnecessary div